### ISSUE TITLE
Cookie's Bustle Busts 86-Box

### ISSUE THREAD

**Issue by ualaa1** [86Box/86Box#3195]
**Describe the bug**
Game crashes 86-Box immediately after opening

**To Reproduce**
Install Quicktime 6.52 first. Then load the CD of the game. Go into the directory of the CD and load setup.exe
There will be Mojibake on the installation screen. Just keep going. Later in the installation process, error messages will pop up saying that the game can't be installed on the start menu. Press enter to these messages. Game will finish installing. Go to C:\Programs\cb1
Drag out CB1W out to the desktop to create a shortcut. Click on shortcut. Game will ask for a registration number. The registration number of the copy I have is CB1W-30003-1007-66
After typing it, game will automatically close 86-box. On subsequent attempts to open it, game immediately causes a crash
I have tried adjusting the color settings and speed and the game still crashes

**Expected behavior**
For game to play

**Screenshots**

**Desktop (please complete the following information):**
Windows 10 3.11 x64

**Additional context**
NVR: 
[nvr.zip](https://github.com/86Box/86Box/files/10947573/nvr.zip)

CFG:
[86box.zip](https://github.com/86Box/86Box/files/10947575/86box.zip)

**Comment by pankoza-pl** [comment]
try a 32-bit build of 86Box if you are using a 64-bit one
64-bit builds are currently buggy

**Comment by cold-brewed** [comment]
Have you tried different video card and system combinations? I'm not sure what the guest OS is, but if you require voodoo I'd suggest perhaps trying a voodoo 1000/2000/3000 first. 

I'd also suggest dialing the guest specs back a bit: Instead of using a down-clocked socket 370 Mendocino, try an earlier board such as socket 7 and a regular pentium. It's hard to say without knowing the specs of your host system.

Make sure to use a fresh install of 9x on the new guest.

**Comment by ualaa1** [comment]
I was using Windows 98 SE with a Voodoo 3500 SE (two threads) and a Mendocino at 133 mhz.
I was running this on a i7-9750H

**Comment by ualaa1** [comment]
Figured out how to get the game working for now. I tried all sorts of fresh installs on different settings and it wasn't working. While I still think the game outright crashing 86-Box is still a bug, I managed to find a workaround preventing the crash.
Go into the Quicktime player video settings and unchecking the directdraw acceleration seems to work. I have played the game and so far no crashes that I can see.
